examples/embed/example_vertexai.py

```diff
@@ -0,0 +1,24 @@
+from unstructured.documents.elements import Text
+from unstructured.embed.vertexai import VertexAIEmbeddingConfig, VertextAIEmbeddingEncoder
+
+# https://python.langchain.com/docs/integrations/text_embedding/google_vertex_ai_palm
+# To use Vertex AI PaLM you must either have credentials configured for your environment (gcloud,
+# workload identity, etc…), or store the path to a service account JSON file as the
+# GOOGLE_APPLICATION_CREDENTIALS environment variable.
+
+# Or, you can pass the json content of your API key to the VertexAIEmbeddingConfig
+# like this: VertexAIEmbeddingConfig(api_key_json=GOOGLE_APPLICATION_CREDENTIALS_JSON_CONTENT)
```

Comment:
api_key not api_key_json

Reply:
[working example in examples with comments updated](https://github.com/Unstructured-IO/unstructured/pull/2693/commits/5b278bf93fe0fd08eb0a97ed66485f049c960d6c)